micro-benchmarks/DRB188-bar3_no.c

```diff
@@ -0,0 +1,117 @@
+/*
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+!!! Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+!!! and DataRaceBench project contributors. See the DataRaceBench/COPYRIGHT file for details.
+!!!
+!!! SPDX-License-Identifier: (BSD-3-Clause)
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+ */
+
+/*
+ * This is a program based on a dataset contributed by 
+ * Wenhao Wu and Stephen F. Siegel @Univ. of Delaware.
+ 
+ * implements 2-thread reuseable barrier using 3 locks, no race [inline]
```

Comment:
What does [inline] mean here? @LChenGit

Reply:
It is from the original repo. I have removed it.